### PR TITLE
updated README with latest changes

### LINKED ISSUE #50: update / verify README.md reflects the latest changes

### PR DESCRIPTION
**Pull request by dericcrago**
closes #50

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,46 +38,54 @@ Along with the `main` branch, there is currently a branch for each major Ansible
 │   ├── latest_builds.py
 │   ├── matrix.yml
 │   └── requirements.txt
+└── test
+    ├── README.md
+    ├── main.sh
+    ├── script.lib
+    ├── test_vault.yml
+    └── vault_pass
```
Comment (anweshadas) [inline README.md:46]: :+1:

File: README.md [hunk 2]
```diff
@@ -35,46 +38,54 @@ Along with the `main` branch, there is currently a branch for each major Ansible
...
+│   └── packaging
+│       ├── debian
+│       │   ├── control
+│       │   ├── copyright
+│       │   ├── copyright.license
+│       │   ├── rules
+│       │   ├── rules.license
+│       │   └── source
+│       └── templates
+│           └── changelog
```
Comment (anweshadas) [inline README.md:75]: :+1:

File: README.md [hunk 3]
```diff
@@ -35,46 +38,54 @@ Along with the `main` branch, there is currently a branch for each major Ansible
...
+│       │   ├── ansible-core.dirs
+│       │   ├── ansible-core.install
+│       │   ├── control
+│       │   ├── copyright
+│       │   ├── copyright.license
+│       │   ├── rules
+│       │   ├── rules.license
+│       │   └── source
+│       └── templates
+│           └── changelog
```
Comment (anweshadas) [inline README.md:88]: :+1:

**Review by anweshadas — APPROVED** [review]
Looks all good to me.

Approved.

**Review by cybette — APPROVED** [review]

**Pull request merged**